libethereum/State.h

```diff
@@ -336,7 +336,7 @@ class State
 	u256 m_accountStartNonce;
 
 	friend std::ostream& operator<<(std::ostream& _out, State const& _s);
-	std::vector<detail::Change> m_changeLog;
+	std::vector<detail::Change> m_changeLog;	///< Working changeLog
```

Comment:
1. Working means the one that is not broken. You mean current, or change log of currently executing transaction?
2. "changeLog" is not an English word. You mean "change log"?
3. Missing full stop.